SmartDeviceLink/SDLPermissionManager.m

```diff
@@ -149,9 +177,18 @@ - (SDLPermissionObserverIdentifier)addObserverForRPCs:(NSArray<SDLPermissionRPCN
     return filter.identifier;
 }
 
+- (SDLPermissionObserverIdentifier)subscribeToRPCs:(NSArray<SDLRPCFunctionName> *)rpcNames groupType:(SDLPermissionGroupType)groupType withHandler:(SDLPermissionsChangedHandler)handler {
+    SDLPermissionFilter *filter = [SDLPermissionFilter filterWithRPCNames:rpcNames groupType:groupType observer:handler];
+
+    // Store the filter for later use
+    [self.filters addObject:filter];
+
```

Comment:
This doesn't look right. I think you're now _never_ calling the initial observer, which is not correct. See above in the documentation on this topic.